runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java

```diff
@@ -132,7 +132,7 @@ protected void shutDown() throws Exception
 	}
 
 	@Subscribe
-	public void onGameStateChange(GameStateChanged event)
+	void onGameStateChange(GameStateChanged event)
```

Comment:
for subscribers we decided to use public for consistency (and also kind of makes sense as it is indirectly called as public)

Reply:
No problem, reverted the subscriber visibility changes.